listenbrainz/webserver/templates/user/base.html

```diff
@@ -1,11 +1,22 @@
 {% extends 'base-react.html' %}
 
-{% block title %}User "{{ user.musicbrainz_id }}" - ListenBrainz{% endblock %}
+{% block title %}
+    User "{{ user.musicbrainz_id }}" - ListenBrainz
+{% endblock %}
 
 {% block content %}
   <div id="user-profile">
     <div id="user-page-heading-container"></div>
-
+      <p>
+          See profile on&nbsp;
+          <img
+              src="/static/img/musicbrainz-16.svg"
+              alt="MusicBrainz Logo"
+          />
+          <a href="https://musicbrainz.org/user/{{ user.musicbrainz_id }}">
+              MusicBrainz
+          </a>
+      </p>
```

Comment:
I'm in favor of making this consistent (I have no idea why this was on some pages and not others, bad copy/paste I imagine), but I think there's also room for improvement while we're at it.

As a quick suggestion, We could move that `<p>` block right before the `user-page-heading-container` div, and give it a `float:right` CSS rule. The result looks like this:

![image](https://user-images.githubusercontent.com/6179856/153655143-027bbec8-0b44-474d-be06-1ee2079315be.png)

Thoughts?

The reason for putting it before the username is so that it is place higher up and doesn't interfere with the "Report user" button displayed on other user's page

Reply:
For now I think this is good enough.

Reply:
Yeah, i'm with you. Not great by any means, but "good enough" for now